src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx

```diff
@@ -18,6 +18,13 @@ export const claimExamsDescription = (
         schedule your exam. It’s important that you answer any calls you receive
         after you file a disability claim.
       </p>
+      <p>
+        You can go to your{' '}
+        <a href="/profile" target="_blank" rel="noreferrer">
+          VA.gov profile
+        </a>{' '}
```

Comment:
For internal links, I learned we're supposed to open then in the same tab/window ([doc](https://design.va.gov/content-style-guide/links#linking-to-external-sites)). However, 526 has some odd behavior which prevents the user from easily getting back to the page they were on within the form. In that case, we should update the link text to include `(opens in new tab)`

Reply:
@christinec-fftc thanks so much! I'll do that.